main/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsNetworkSchema.java

```diff
@@ -0,0 +1,15 @@
+package org.mobilitydata.gtfsvalidator.table;
+
+import org.mobilitydata.gtfsvalidator.annotation.*;
+
+@GtfsTable("agency.txt")
+@Required
+public interface GtfsNetworkSchema extends GtfsEntity {
```

Comment:
[suggestion]: Add the network schema in a different PR where we can test it in isolation.